Awesome books: refactor to use JavaScript classes

maov19 opened the pull request:
Following features were added:

- Replaced Javascript objects and arrays with classes.
- Maintained functionality.
- Added CSS styles.

maov19 committed:
Add basic html structure

App-Developer8866 committed:
setup linter

maov19 committed:
Merge branch 'development' of https://github.com/maov19/awesome_books into development

maov19 committed:
Fix html tags

maov19 committed:
Add js class for books

maov19 committed:
Add books class

maov19 committed:
Fix add button issue

maov19 committed:
Fix html tags and sections

maov19 committed:
Add styling to html file

maov19 committed:
Fix add book function

maov19 committed:
Fix styling

hiromataba reviewed (approved):
# Status: Approved :tada: :tada: :tada: 
Hi @maov19 and @App-Developer8866     :wave: :wave: 

![200w](https://user-images.githubusercontent.com/75126481/151663786-7e0aa330-6769-4ab2-a051-3754be515be9.gif)  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

maov19 committed:
Add navigation bar

maov19 committed:
Add contact section

maov19 committed:
Add styling to list and contact section

maov19 committed:
Add style display functions

maov19 committed:
Add date function

maov19 committed:
Add class for date and time

maov19 committed:
Fix styling

maov19 commented:
Following features were added:

- Added navigation bar.
- Added three content sections:
1. Books list.
2. Add book form.
3. Contact info.

- When clicked, only selected areas are displayed.
